website/src/utils/search.ts

```diff
@@ -63,7 +63,13 @@ const getFieldOrColumnVisibilitiesFromQuery = (
         if (field.hideOnSequenceDetailsPage === true) {
             return;
         }
-        visibilities.set(field.name, initiallyVisibleAccessor(field) === true);
+
+        let fieldName = field.name;
+
+        if (field.rangeOverlapSearch) {
+            fieldName = field.rangeOverlapSearch.rangeName;
+        }
+        visibilities.set(fieldName, initiallyVisibleAccessor(field) === true);
```

Comment:
The modifications in this function could be avoided if we force users to uses a `field.name` as the `rangeName`. This can't really be enforced. IMO this can't be avoided because we have to group fields two distinct fields.

Happy for input though if anybody happens to look at the PR already!

Reply:
I think we need some more changes in this general area (appreciate the PR is still a draft): atm Collection date shows up twice on the list of fields to change the visibility of. I'd need to think about exactly the best way to handle this.

Reply:
I think it's just there twice because of the second _Collection date_ `string` field which is just above the range field.

I had a vague feeling that we shouldn't look for Metadata field visiblity, but rather MetadataFilter visibility! But also haven't looked into it more yet. But good to know that you think this area could use some more work! I'll spend some more time on it then (probably monday).

Reply:
> I think it's just there twice because of the second Collection date string field which is just above the range field.

OK! That may well be true - my bad - and in that case it may well not need more work - sorry for my misunderstanding!